charts/argo/templates/workflow-aggregate-roles.yaml

```diff
@@ -1,6 +1,10 @@
 {{- if .Values.createAggregateRoles }}
 apiVersion: rbac.authorization.k8s.io/v1
+{{- if .Values.singleNamespace }}
+kind: Role
+{{ else }}
 kind: ClusterRole
+{{- end }}
```

Comment:
Aggregated roles are always `ClusterRole`: https://kubernetes.io/docs/reference/access-authn-authz/rbac/#aggregated-clusterroles

Instead, maybe provide a boolean option to `includeAggregateClusterRoles` to avoid this failing when installing onto the same cluster twice?

Reply:
Looks like there is already an option for it: `.createAggregateRoles`. Removing the non ClusterRole changes.